app/assets/src/components/visualizations/table/base_table.scss

```diff
@@ -33,6 +34,15 @@
   outline: 0;
   text-align: center;
   text-transform: none;
+
+  .text {
```

Comment:
Can you call it something more clear? e.g. `columnHeaderTooltipText`?

Reply:
sure, although this was already in .header

Reply:
oh, i did not notice that. Could be just `tooltipText` then